### PR TITLE
Login signup page

### PR DESCRIPTION
**Pull request by gadgetfather**
**Screens only**
image

image

### CODE REVIEW

**Review by Tejinder15 — APPROVED** [review]

File: src/index.css [hunk 1]
```diff
@@ -4,3 +4,26 @@
...
+.tint-image {
+  background:  
+    /* top, transparent red, faked with gradient */ linear-gradient(
+      rgba(0, 0, 0, 0.5),
+      rgba(0, 0, 0, 0.5)
+    ),
+    url("https://images.unsplash.com/photo-1542751371-adc38448a05e?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1740&q=80");
+  /* object-fit: cover;
+  object-position: center; */
+  /* background-repeat: no-repeat; */
```
Comment (NamrataRaikwar2002) [inline src/index.css]: You can remove these comments.

File: src/App.js [hunk 2]
```diff
@@ -1,5 +1,12 @@
+import { LoginPage, SignupPage } from "./pages";
+
 function App() {
-  return <div>hello</div>;
+  return (
+    <div>
+      {/* <LoginPage /> */}
```
Comment (NamrataRaikwar2002) [inline src/App.js:6]: Here also

**Review by NamrataRaikwar2002 — APPROVED** [review]
- The UI looks nice
- code is clean 
except for one suggestion, all looks good.
Great work :+1:

File: src/index.css (hunk 1 above)
Comment (gadgetfather) [reply src/index.css]: Yea thanks for pointing out

**Pull request merged**